### PR TITLE
External Documentation rendered for groups, operations and schemata.

### PR DESCRIPTION
**Pull request by m-mohr**
The externalDocs were missing for tags, schemata and operations. Added them with this pull requests. Solves #550.

Additionally, fixes that the URL in External Documentation Object was specified to be optional, which is not correct according to OpenAPI spec.

### CODE REVIEW

File: src/components/ContentItems/ContentItems.tsx [hunk 1]
```diff
@@ -78,6 +79,15 @@ export class SectionItem extends React.Component<ContentItemProps> {
           </MiddlePanel>
         </Row>
         <AdvancedMarkdown source={description || ''} htmlWrap={middlePanelWrap} />
+        {externalDocs && (
+          <Row>
+            <MiddlePanel>
+              <p>
```
Comment (RomanGotsiy) [inline src/components/ContentItems/ContentItems.tsx]: Do we really need a `<p>` here?

File: src/components/Operation/Operation.tsx [hunk 2]
```diff
@@ -50,6 +51,11 @@ export class Operation extends React.Component<OperationProps> {
               </H2>
               {options.pathInMiddlePanel && <Endpoint operation={operation} inverted={true} />}
               {description !== undefined && <Description source={description} />}
+              {externalDocs && (
+                <p>
```
Comment (RomanGotsiy) [inline src/components/Operation/Operation.tsx]: Same here

File: src/components/ContentItems/ContentItems.tsx (hunk 1 above)
Comment (m-mohr) [reply src/components/ContentItems/ContentItems.tsx]: That's on purpose. The p adds a margin between the External Documentation link and the description. Without the margin in looks odd. The div is chosen in the schema as it seems to be designed to be very small and therefore without margin, like most other elements in there, too. So I think your suggestion makes things look worse without introducing additional styling per place that it's added.

File: src/components/Operation/Operation.tsx (hunk 2 above)
Comment (m-mohr) [reply src/components/Operation/Operation.tsx]: See above.

**Comment by RomanGotsiy** [comment]
I cleaned it up a bit myself!

Thanks for the PR.

It will land in the upcoming alpha at the end of the week.

**Comment by m-mohr** [comment]
Thank you!

**Pull request merged**